O/ONNXRuntime/build_tarballs.jl

```diff
@@ -115,18 +125,44 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("CUDNN_jll", cudnn_version;
-        compat = cudnn_compat,
-        platforms = cuda_platforms),
-    Dependency("TensorRT_jll", tensorrt_version;
-        compat = tensorrt_compat,
-        platforms = cuda_platforms),
-    Dependency("Zlib_jll"; platforms = cuda_platforms),
-    HostBuildDependency(PackageSpec("protoc_jll", Base.UUID("c7845625-083e-5bbe-8504-b32d602b7110"), v"3.16.1"))
+    HostBuildDependency(PackageSpec("protoc_jll", v"3.16.1"))
 ]
 
-# Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-    lazy_artifacts = true,
-    julia_compat = "1.6",
-    preferred_gcc_version = v"8")
+builds = []
+for platform in platforms
+    should_build_platform(platform) || continue
+    additional_deps = BinaryBuilder.AbstractDependency[]
+    if haskey(platform, "cuda") && platform["cuda"] != "none"
+        if platform["cuda"] == "11.3"
+            additional_deps = BinaryBuilder.AbstractDependency[
+                BuildDependency(PackageSpec("CUDA_full_jll", v"11.3.1")),
+                Dependency("CUDA_Runtime_jll", v"0.7.0"), # Using v"0.7.0" to get support for cuda = "11.3" - using Dependency rather than RuntimeDependency to be sure to pass audit
+            ]
+        else
```

Comment:
I'm not sure I like this, we pretty much decided we don't support 11.3 anymore. Why can't you bump TensorRT so that it supports a reasonable CUDA version?

Reply:
… and to get changes done in ONNXRuntime.jl (which will also require an update to a modern version to get PRs through).

Reply:
… and verify that TensorRT works before pushing hundreds of MB to JuliaBinaryWrappers.